pkg/ccl/cliccl/load_test.go

```diff
@@ -207,3 +195,24 @@ func checkExpectedOutput(t *testing.T, expected string, out string) {
 	output := out[endOfCmd+1:]
 	require.Equal(t, expected, output)
 }
+
+// Generate n continuous TimeStamps with minimal interval of 10 milliseconds to be used in incremental backup tests.
+func generateContinuousTimeStamps(n int) []hlc.Timestamp {
+	timeStamps := make([]hlc.Timestamp, 0, n)
+	if n == 0 {
+		return timeStamps
+	}
+	lastTS := hlc.Timestamp{WallTime: timeutil.Now().UnixNano()}
+	timeStamps = append(timeStamps, lastTS)
+	for {
```

Comment:
nit: Have you considered the approach of doing a simple loop and just sleeping between the iterations? That looks like it would do what we want and may be easier to reason through:

```
for i := 0; i < n; i++ {
    timestamps = append(timestamps, hlc.Timestamp{WallTime: timeutil.Now().UnixNano()})
    time.sleep(10 * time.Millisecond)
}
```

Reply:
yea.. much more easier. updated